### DISCUSSION TITLE
CPP Whisper Large Model

### DISCUSSION THREAD

**Discussion by JustAGuyM** [SubtitleEdit/subtitleedit#7841] [Q&A] [answered]
Hello, what is the difference between the 'large-v1,' 'large-v2,' 'large,' and 'large-q5_0' models? Is there one that is the best and most recommended? I want to use it to recognize the Japanese language. Thanks in advance. I'm using 4.03 beta.
![models](https://github.com/SubtitleEdit/subtitleedit/assets/140866753/fefe3aee-cbfe-4985-bde6-617caa3579be)

**Comment by niksedk** [comment, marked as answer]
The `large-v1` model is from the initial version of Whisper from OpenAI (Sep, 2022).

The `large-v2` model is a newer version of the large model (Dec, 2022):
https://github.com/openai/whisper/discussions/661

The  `large` / `large-v3` model is the latest version of the large model (Nov, 2023):
https://github.com/openai/whisper/discussions/1762

The `large-q5_0` is a smaller version of the `large-v2` 
https://github.com/ggerganov/whisper.cpp/discussions/1414

Note: Training these models have probably taken many weeks and costed several million dollars I suspect.
